Description: Disable hpet for unaccelerated qemu
 Hpet is not really needed for unaccelerated qemu, and currently causes it
 to hang on boot.  The bug is fixed upstream, but the fix is not cleanly
 cherrypick-able, so simply "don't do that".
Author: Serge Hallyn <serge.hallyn@ubuntu.com>
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=668186
Bug-Ubuntu: https://bugs.launchpad.net/debian/+source/qemu-kvm/+bug/975240
Forwarded: no

Index: qemu-kvm-1.0+noroms/vl.c
===================================================================
--- qemu-kvm-1.0+noroms.orig/vl.c	2012-04-09 11:03:55.691027742 -0500
+++ qemu-kvm-1.0+noroms/vl.c	2012-04-09 11:05:00.287348049 -0500
@@ -2063,6 +2063,10 @@
         exit(1);
     }
 
+    /* for non-accelerated qemu, don't enable hpet */
+    if (!strcmp(accel_list[i].opt_name, "tcg"))
+        no_hpet = 1;
+
     if (init_failed) {
         fprintf(stderr, "Back to %s accelerator.\n", accel_list[i].name);
     }
